readmes/README_ja_JP.md

```diff
@@ -0,0 +1,1126 @@
+[![ビルド
+ステータス](https://travis-ci.com/puppetlabs/puppetlabs-azure.svg?token=RqtxRv25TsPVz69Qso5L)](https://travis-ci.com/puppetlabs/puppetlabs-azure)
+
```

Comment:
confusing its pointing to travis (https://travis-ci.com/puppetlabs/puppetlabs-azure.svg?token=RqtxRv25TsPVz69Qso5L)](https://travis-ci.com/puppetlabs/puppetlabs-azure)